pkg/apiserver/info/info.go

```diff
@@ -61,11 +77,35 @@ type InfoResponse struct { //nolint
 // @Security JwtAuth
 // @Failure 401 {object} rest.ErrorResponse
 func (s *Service) infoHandler(c *gin.Context) {
+	// Checking ngm deployments
+	// drop "-alpha-xxx" suffix
+	versionWithoutSuffix := strings.Split(s.params.Config.FeatureVersion, "-")[0]
+	v, err := semver.NewVersion(versionWithoutSuffix)
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+	constraint, err := semver.NewConstraint(">= v5.4.0")
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+
+	ngmState := utils.NgmStateNotSupported
+	if constraint.Check(v) {
+		ngmState = utils.NgmStateNotStarted
+		addr, err := topology.FetchNgMonitoringTopology(s.lifecycleCtx, s.params.EtcdClient)
```

Comment:
Could you confirm the behavior when etcd is down?

Reply:
When etcd is down, ngm proxy may not work. So we can't use ngm and conprof.